src/parallel/bbs.cpp

```diff
@@ -294,12 +284,11 @@ void BBSImpl::execute(int id) {  // assumes a "_todo" message in receive buffer
     pkbegin();
     pkint(userid);
     pkint(wid);
-    pkint(rs ? 1 : 0);
-    if (!rs) {
+    pkint(!rs.empty() ? 1 : 0);
+    if (rs.empty()) {
```

Comment:
This is where we might need the difference between `nullptr` and `""`.

Reply:
a valid pickle cannot be empty